src/js/components/stock-movement-wizard/combined-shipments/AddItemsPage.jsx

```diff
@@ -505,7 +544,7 @@ class AddItemsPage extends Component {
    * @public
    */
   fetchLineItems() {
-    const url = `/api/stockMovements/${this.state.values.stockMovementId}/stockMovementItems?stepNumber=2`;
+    const url = `/openboxes/api/stockMovements/${this.state.values.stockMovementId}/stockMovementItems?stepNumber=2`;
```

Comment:
I see there are a lot of URL that you have updated (added context _/openboxes_).
Since there is no time pressure to release this fix, maybe you can move these URLs to `api/url.js`.

It could also be done in some separate spike ticket, but if you can then please do it.

Reply:
@drodzewicz @alannadolny If this didn't get done we should create a tech debt ticket for any remaining issues. Or we should keep the "replace context path" ticket open until we're ready for testing.

Reply:
@jmiranda I did it in this PR